modules/dreamview/backend/teleop/README.md

```diff
@@ -0,0 +1,27 @@
+
+# Dreamview for Teleop
+
+## Introduction
+Dreamview module provides a web application that helps operate and visualize teleop feature.
+
+## Teleop Services
+
+### Send Audio Stream
+
+### Send Mic Stream
+
+### Send Video Stream
+
+### Send Emergency Stop Command
+- When "Stop" button is clicked, a pad message with `DrivingAction` of `STOP`, defined by Protobuf message `PadMessage` in file `planning/proto/pad_msg.proto`, will be puslished by **Dreamview**.
+- Upon receiving a pad message with `DrivingAction` of `STOP`, **Planning** module will execute **emergency stop** by abruptly brake and stop at the current route.
+
+### Send Emergency Pull-over Command
+- When "Pull OVer" button is clicked, a pad message with `DrivingAction` of `PULL_OVER`, defined by Protobuf message `PadMessage` in file `planning/proto/pad_msg.proto`, will be puslished by **Dreamview**.
```

Comment:
Pull OVer -> Pull Over

Reply:
Maybe "by" should be followed by "[verb]-ing" ?